### PR TITLE
Tommy garch patch 1

### PR DESCRIPTION
**Pull request by TommyGarch**

### CODE REVIEW

**Review by jiajames — APPROVED** [review]

File: pages/Architecture/indexer.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Indexer Deep Dive
+
+A good way to think about the indexer is as similar to Infura or Alchemy’s role in the Ethereum ecosystem. However, unlike Infura/Alchemy, and like everything else in dYdX v4, the Indexer is completely open source and can be run by anyone!
```
Comment (danlasko) [inline pages/Architecture/indexer.md:3]: Change "in dYdX v4" to "with dYdX Chain"

File: pages/Architecture/indexer.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+### What is the Indexer?
+
+As part of tooling for the dYdX ecosystem, we want to ensure that clients have access to performant data queries when using the dYdX v4 exchange. Cosmos SDK Full Nodes offer a number of APIs that can be used to request on-chain data. However, these Full Nodes are optimized for committing and executing blocks, not for serving high frequency, low-latency requests from web/mobile clients.
```
Comment (danlasko) [inline pages/Architecture/indexer.md:7]: Replace "when using dYdX v4 exchange" with "when using exchanges running on dYdX Chain software"

File: pages/Architecture/indexer.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+### Comlink (API Server)
+
+Comlink is an API server that will expose REST API endpoints to read both on-chain and off-chain data. For example, a user could request their USDC balance or the size of a particular position through Comlink, and would receive a formatted JSON response.
+
+As an explicit goal set out by the dYdX team, we’re designing v4 APIs to closely match the [v3 exchange APIs](https://dydx.exchange/blog/v4-deep-dive-indexer#:~:text=closely%20match%20the-,v3%20exchange%20APIs,-.%20We%20have%20had). We have had time to gather feedback and iterate on these APIs over time with v3, and have confidence that they are reasonable at the product-level.
```
Comment (danlasko) [inline pages/Architecture/indexer.md:58]: Delete "exchange" in "v3 exchange APIs"

File: pages/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# FAQ Documentation
+
+Since the inception of v4, we have been compiling a list of relevant questions and answers that are asked by MMs/Trading Firms on Telegram. We turned this list into a live _v4 Telegram Questions_ document, which should be able to answer many of your more nuanced questions.
```
Comment (danlasko) [inline pages/FAQ.md:3]: Replace "v4" with "dYdX Chain" , change "relevant questions and answers that are asked by MMs/Trading Firms on Telegram," to "questions and answers from users related to technical support" - take "v4 Telegram" out of doc title and call it "User Support Questions" or something

File: pages/guides/how_to_interpret_block_data_for_trades.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+## dYdX v4: How to Interpret the Block Data for Trades
+
+<img width="639" alt="Screenshot 2024-03-29 at 2 34 06 PM" src="https://github.com/dydxprotocol/v4-documentation/assets/130097657/19bc0173-0d79-473f-b88a-f99b72c7065a">
+
+In dYdX trading, quantities and prices are represented in quantums (for quantities) and subticks (for prices), which need conversion for practical understanding.
```
Comment (danlasko) [inline pages/guides/how_to_interpret_block_data_for_trades.md:5]: Add "Chain" after dYdX

File: pages/guides/how_to_interpret_block_data_for_trades.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+If BTC drops to 200 USDC/BTC, a `tick` being 100 USDC/BTC no longer makes sense, and we may want a `tick` to be 1 USDC/BTC, which lets us set `subtick_per_tick` to 100 to get to a `tick` size of 1 USDC/BTC.
+
+### Now back to the interpretation of the above image:
+
+1. First, notice column I is negative.  That means this trade is a sell by the taker account.  If It was positive, it would be a buy.
+
+Result: Determined if this is a buy or a sell
+
+2. Next, look at column N.  The perpetual_id is 7, which maps to AVAX-USD market.  You can see all the mappings from this endpoint https://indexer.dydx.trade/v4/perpetualMarkets where the clobPairId is the perpetual_id.
```
Comment (danlasko) [inline pages/guides/how_to_interpret_block_data_for_trades.md:31]: Is this the mapping for the DOS mainnet?  If so would add "for the dYdX Chain deployment by dYdX Operations Services Ltd." after "all of the mappings"

File: pages/FAQ.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# FAQ Documentation
+
+Since the inception of dYdX Chain, we have been compiling a list of questions and answers from users related to technical support. We turned this list into a live _v4 Telegram Questions_ document, which should be able to answer many of your more nuanced questions.
```
Comment (danlasko) [inline pages/FAQ.md:3]: Change "Telegram" to "User Support"

**Pull request merged**